packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -65,10 +66,32 @@ static final class MethodNames {
     static final String IS_FEATURE_SUPPORTED = "BillingClient#isFeatureSupported(String)";
     static final String GET_CONNECTION_STATE = "BillingClient#getConnectionState()";
     static final String GET_BILLING_CONFIG = "BillingClient#getBillingConfig()";
+    static final String IS_ALTERNATIVE_BILLING_ONLY_AVAILABLE =
+        "BillingClient#isAlternativeBillingOnlyAvailable()";
+    static final String CREATE_ALTERNATIVE_BILLING_ONLY_REPORTING_DETAILS =
+        "BillingClient#createAlternativeBillingOnlyReportingDetails()";
+    static final String SHOW_ALTERNATIVE_BILLING_ONLY_INFORMATION_DIALOG =
+        "BillingClient#showAlternativeBillingOnlyInformationDialog()";
 
     private MethodNames() {}
   }
 
+  @VisibleForTesting
+  static final class MethodArgs {
+
+    // Key for an int argument passed into startConnection
+    static final String HANDLE = "handle";
+    // Key for a boolean argument passed into startConnection.
+    static final String BILLING_CHOICE_MODE = "billingChoiceMode";
+
+    private MethodArgs() {}
+  }
+
+  static final class BillingChoiceMode {
```

Comment:
Leave a comment here saying the values must match the BillingChoiceMode enum on the Dart side?

Reply:
Good idea, done